shop/models/fields.py

```diff
@@ -12,6 +12,8 @@
     import psycopg2
 
     version = re.search('([0-9.]+)', psycopg2.__version__ or "0").group(0)
+    # To be able to use the Django version of JSONField, it requires to have PostgreSQL ≥ 9.4 , otherwise some issues
+    #  could be faced.
     if POSTGRES_FLAG and (LooseVersion(version) >= LooseVersion('2.5.4')):
```

Comment:
This looks like it checks the psycopg version, not the version of Postgres itself. Or are these two somehow related?